src/test/java/com/microsoft/sqlserver/jdbc/unit/statement/StatementTest.java

```diff
@@ -1111,20 +1111,20 @@ public void testJdbc41CallableStatementMethods() throws Exception {
                                 + "(?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?)}")) {
                     cstmt.registerOutParameter(1, java.sql.Types.VARCHAR);
                     cstmt.registerOutParameter(2, java.sql.Types.INTEGER);
-                    cstmt.registerOutParameter(3, java.sql.Types.FLOAT);
-                    cstmt.registerOutParameter(4, java.sql.Types.DECIMAL);
+                    cstmt.registerOutParameter("col3Value", java.sql.Types.FLOAT);
+                    cstmt.registerOutParameter(4, java.sql.Types.DECIMAL, 2);
                     cstmt.registerOutParameter(5, microsoft.sql.Types.GUID);
                     cstmt.registerOutParameter(6, java.sql.Types.SQLXML);
-                    cstmt.registerOutParameter(7, java.sql.Types.VARBINARY);
+                    cstmt.registerOutParameter("col7Value", java.sql.Types.VARBINARY, 10);
```

Comment:
Reason for changes in this test?

Reply:
for coverage, calls other registerOutParameter with different args